docs/download.md

```diff
@@ -177,8 +177,8 @@ You can bypass this by clicking "More info" then "Run anyway".
 
 |                                                   Package                                                   |         Type          |
 | :---------------------------------------------------------------------------------------------------------: | :-------------------: |
-| [Setup](https://github.com/pulsar-edit/pulsar/releases/download/v1.106.0/Windows.Pulsar.Setup.1.106.0.exe)  |       Installer       |
-| [Portable](https://github.com/pulsar-edit/pulsar/releases/download/v1.106.0/Windows.Pulsar-1.106.0-win.zip) | Portable (no install) |
+| [Setup](https://github.com/pulsar-edit/pulsar/releases/download/v1.107.0/Windows.Pulsar.Setup.1.107.0.exe)  |       Installer       |
+| [Portable](https://github.com/pulsar-edit/pulsar/releases/download/v1.107.0/Windows.Pulsar-1.107.0-win.zip) | Portable (no install) |
```

Comment:
```suggestion
| [Zip](https://github.com/pulsar-edit/pulsar/releases/download/v1.107.0/Windows.Pulsar-1.107.0-win.zip) | Zip (no install) |
```

Does anyone mind if I update this??

"Portable" --> "Zip", since we changed the file format a bit ago?

Reply:
Ooooops this PR is merged already, I can do it elsewhere in another tiny little PR, I suppose.